### DISCUSSION TITLE
Wifi with Dual-Q 5G2PHY

### DISCUSSION THREAD

**Discussion by rmarlan** [iamromulan/cellular-modem-wiki#77] [Ideas/Requests - Quectel]
I have a Dual-Q 5G2PHY with a Quectel RM520N-GL.  Works perfectly with all iamromulan options!  Thank you iamromulan.

Now if I travel, I would like to have wifi.  Of course, I could use a small "travel router", but is there any Wifi dongle that would work via USB or USB to go.  There are some dongles that work with Linux, but I think the dongle would have to work with android and have the correct drivers.   Any ideas???

Thanks for a great product.

Bob

**Comment by iamromulan** [comment]
Something like this would probably require access to the kernel source code of the target modem. What I can tell you though is we have a choice between Ethernet OR WiFi. I'll have to look up where I found that info. I think it was in the PDF for the Quectel M.2 5g dev board. (Which is the official board we can use to compare with for QoS against other Ethernet boards)

**Comment by iamromulan** [comment]
This is something I'd love to see as well though. I'll look into it. The USB port has to allow endpoint devices first.
